subprojects/docs/src/docs/userguide/dependencyManagementTerminology.adoc

```diff
@@ -0,0 +1,49 @@
+[[dependency_management_terminology]]
+== Dependency Management Terminology
+
+Dependency management comes with a wealth of terminology. Here you can find the most commonly-used terms including references to the user guide to learn about their practical application.
+
+[[sub:terminology_configuration]]
+=== Configuration
+
+A configuration is a named set of <<sub:terminology_dependency,dependencies>> grouped together for a specific goal: For example the implementation configuration represents the set of dependencies required to compile a project. Configurations provide access to the underlying, resolved <<sub:terminology_module,modules>> and their artifacts. For more information, see <<sub:scope_of_dependency_configurations>>.
+
+[NOTE]
+====
+The word "configuration" is an overloaded term and has a different meaning outside of the context of dependency management.
+====
+
+[[sub:terminology_dependency]]
+=== Dependency
+
+A dependency is another piece of software required to build, test or run a <<sub:terminology_module,module>>. For more information, see <<declaring_dependencies>>.
+
+[[sub:terminology_module]]
+=== Module
+
+A piece of software that evolves over time e.g. link:https://github.com/google/guava[Google Guava]. Every module has a name. Each release of a module is optimally represented by a <<sub:terminology_module_version,module version>>. For convenient consumption, modules can be hosted in a <<sub:terminology_repository,repository>>.
+
+[[sub:terminology_module_metadata]]
+=== Module metadata
+
+Releases of a <<sub:terminology_module,module>> can provide metadata. Metadata is the data that describes the module in more detail e.g. the coordinates for locating it in a repository, information about the project or required <<sub:terminology_transitive_dependency,transitive dependencies>>. In Maven the metadata file is called `.pom`, in Ivy it is called `ivy.xml`.
+
```

Comment:
I would mention Gradle metadata. Imo we have to start integrating it into the docs wherever it fits even if it is not yet stable.

E.g. add a sentence: _Gradle also supports its own richer metadata in `.module` files._

Reply:
I'd like to see this being added by a pull request that explains `.module` files in general. I think we should keep changes together that belong together. Just mentioning `.module` files without giving a reader further references might pose more questions than answers.